Sources/_SwiftSyntaxMacros/Macro.swift

```diff
@@ -39,3 +45,16 @@ extension Macro {
   /// Default, empty documentation string for macros.
   public static var documentation: String { "" }
 }
+
+#if canImport(_CompilerPluginSupport)
+extension Macro {
+  public static func _name() -> (UnsafePointer<UInt8>, count: Int) {
```

Comment:
Makes sense! I assume we'll do the same for `documentation`, `signature`, and `genericSignature`?